### ISSUE TITLE
Invalid call to add_raw_postfix

### ISSUE THREAD

**Issue by cytadela8** [jertel/elastalert2#1127]
In elastalert:get_hits_terms there are invalid calls to `add_raw_postfix` without required argument `is_five_or_above`.

Problematic call:

https://github.com/jertel/elastalert2/blob/48e875a34bc2f964a423d661f2bbb003d219e8dd/elastalert/elastalert.py#L471

Example rule triggering Exception:

```
 name: Test rule name
    alert_subject: "Test rule subject at {0}"
    alert_subject_args: 
      - "@timestamp"
    alert_text_type: exclude_fields
    type: frequency
    timeframe:
      minutes: 10
    num_events: 1
    index: logs-*
    alert:
    - "slack"
    filter:
    - query:
        query_string:
          query: "REDACTED"
    query_key:
    - "labels.k8s-cluster"
```

ElastAlert2 logs with exception:
```
Reading Elastic 8 index mappings:
Reading index mapping 'es_mappings/8/silence.json'
Reading index mapping 'es_mappings/8/elastalert_status.json'
Reading index mapping 'es_mappings/8/elastalert.json'
Reading index mapping 'es_mappings/8/past_elastalert.json'
Reading index mapping 'es_mappings/8/elastalert_error.json'
Index elastalert already exists. Skipping index creation.
ERROR:elastalert:Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/elastalert/elastalert.py", line 1299, in alert
    return self.send_alert(matches, rule, alert_time=alert_time, retried=retried)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/elastalert/elastalert.py", line 1334, in send_alert
    counts = self.get_top_counts(rule, start, end, keys, qk=qk)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/elastalert/elastalert.py", line 1807, in get_top_counts
    hits_terms = self.get_hits_terms(rule, starttime, endtime, index, key, qk, number)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/elastalert/elastalert.py", line 471, in get_hits_terms
    filter_key = add_raw_postfix(filter_key)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: add_raw_postfix() missing 1 required positional argument: 'is_five_or_above'

ERROR:elastalert:Uncaught exception running rule Test rule name: add_raw_postfix() missing 1 required positional argument: 'is_five_or_above'
ERROR:elastalert:Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/elastalert/elastalert.py", line 1299, in alert
    return self.send_alert(matches, rule, alert_time=alert_time, retried=retried)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/elastalert/elastalert.py", line 1334, in send_alert
    counts = self.get_top_counts(rule, start, end, keys, qk=qk)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/elastalert/elastalert.py", line 1807, in get_top_counts
    hits_terms = self.get_hits_terms(rule, starttime, endtime, index, key, qk, number)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/elastalert/elastalert.py", line 471, in get_hits_terms
    filter_key = add_raw_postfix(filter_key)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: add_raw_postfix() missing 1 required positional argument: 'is_five_or_above'

ERROR:elastalert:Uncaught exception running rule Test rule name: add_raw_postfix() missing 1 required positional argument: 'is_five_or_above'
```

**Comment by jertel** [comment]
Thanks for reporting this. Your test rule needs to have `top_count_keys` defined in order to reproduce this. I will submit a PR later today.

**Comment by jertel** [comment]
Fixed in #1128
